src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Verbs.tt

```diff
@@ -0,0 +1,213 @@
+<#@ template debug="false" hostspecific="true" language="C#" #>
+<#@ assembly name="System.Xml.XDocument" #>
+<#@ assembly name="System.Xml.Linq" #>
+<#@ assembly name="System.Xml" #>
+<#@ assembly name="System.Core" #>
+<#@ import namespace="System.Xml.Linq" #>
+<#@ import namespace="System.Linq" #>
+<#@ output extension=".cs" #>
+<#
+
+ string InitCaps(string input)
```

Comment:
yes, will go share that code better.

Reply:
i tried this, but i'm bailing as the include functionality of T4 was taking a bit too long. Wasn't impressed with the ease of use. Consolidated so that all 3 .tt have identical helpers for now.